tests/appsec/iast/test_iast_sql_injection.py

```diff
@@ -2,11 +2,26 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/).
 # Copyright 2021 Datadog, Inc.
 
-from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released
+from utils import BaseTestCase, interfaces, context, missing_feature, coverage, released, bug
 
 # Weblog are ok for nodejs/express4 and java/spring-boot
 @coverage.basic
-@released(dotnet="?", golang="?", java="?", nodejs="?", php_appsec="?", python="?", ruby="?", cpp="?")
+@released(
+    dotnet="?",
+    golang="?",
+    java={
+        "spring-boot": "0.108.0",
```

Comment:
These versions for SQL Injection are wrong. They should be 1.1.0. Anyway, I changed this here: https://github.com/DataDog/system-tests/pull/681/files